src/api/public/apidocs-new/paths/source_project_name_cmd_undelete.yaml

```diff
@@ -0,0 +1,46 @@
+post:
+  summary: Restore a removed project.
+  description: |
+    If the given project was deleted previously, you can bring it back.
+    You can only restore it if you have the permissions to do so.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+  responses:
+    '204':
+      description: |
+        No Content.
+
+        When the `undelete` is correctly performed, there isn't an OK response but this 204 one.
```

Comment:
please do not document bugs....

Reply:
Alright, I can quickly fix this response with this PR https://github.com/openSUSE/open-build-service/pull/11932, so I can document the correct one.

I also created another PR based on the previous one to propose a different approach for the error response: https://github.com/openSUSE/open-build-service/pull/11933

I've split them into two PRs just in case you have any concerns about the error part, we can at least merge the success one and go on with the API documentation of this endpoint.

Please have a look, @adrianschroeter